functions/getMessages.ts

```diff
@@ -0,0 +1,59 @@
+import '@twilio-labs/serverless-runtime-types';
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+  TwilioResponse,
+} from '@twilio-labs/serverless-runtime-types/types';
+
+const TokenValidator = require('twilio-flex-token-validator').functionValidator;
+
+// TODO: Factor out into lib
+const send = (statusCode: number) => (body: string | object) => (callback: ServerlessCallback) => (
+  response: TwilioResponse,
+) => {
+  response.setStatusCode(statusCode);
+  response.setBody(body);
+  callback(null, response);
+};
+
+// TODO: Factor out into lib
+const responseWithCors = () => {
+  const response = new Twilio.Response();
+
+  response.setHeaders({
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Methods': 'OPTIONS, POST, GET',
+    'Access-Control-Allow-Headers': 'Content-Type',
+    'Content-Type': 'application/json',
+  });
+
+  return response;
+};
+
+type Body = {
+  language: string | undefined;
```

Comment:
Can we use the `?` operator here: `language?: string`
(and other parts of the code with similar `| undefined` definitions)

Reply:
Yes. I was using the `| undefined` just to be more explicit, but if you are ok with the optional  operator, I'll change to that!